app/components/WebsocketProvider.tsx

```diff
@@ -464,10 +463,7 @@ class WebsocketProvider extends React.Component<Props> {
 
     this.socket.on("users.demote", async (event: PartialWithId<User>) => {
       if (auth.user && event.id === auth.user.id) {
-        await Promise.all([
-          documents.fetchPage({ limit: Pagination.defaultLimit }),
-          collections.fetchPage({ limit: Pagination.defaultLimit }),
-        ]);
+        await Promise.all([documents.fetchAll(), collections.fetchAll()]);
```

Comment:
Lets just remove the document policies and treat it differently for now, doesn't seem like a good idea to pull all documents down

Reply:
You know, that's a problem when we change back from viewer to member. They'd have to refresh in this case to be able to edit the document again. But, in a way this might be _ok_.

Reply:
Yea, I think you could easily add the same frontend handling for `users.promote`

Reply:
Looks like the role change from _viewer_ to _member_ still happens through `users.demote`, despite it being a _promotion_ semantically. `users.promote` is hit when the permission is upgraded to _admin_. Can tackle this separately later.